OutOfSchool/OutOfSchool.EmailSender/Services/EmailSenderService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using OutOfSchool.EmailSender.Quartz;
+using Quartz;
+
+namespace OutOfSchool.EmailSender.Services;
+
+public class EmailSenderService : IEmailSenderService
+{
+    private readonly IScheduler scheduler;
+
+    public EmailSenderService(IScheduler scheduler)
+    {
+        this.scheduler = scheduler;
+    }
+
+    public async Task SendAsync(string email, string subject, (string html, string plain) content, DateTime? expirationTime = null)
+    {
+        expirationTime ??= DateTime.MaxValue;
+
+        var jobData = new JobDataMap
+        {
+            { EmailSenderStringConstants.Email, email },
+            { EmailSenderStringConstants.Subject, subject },
+            { EmailSenderStringConstants.HtmlContent, EncodeToBase64(content.html) },
+            { EmailSenderStringConstants.PlainContent, EncodeToBase64(content.plain) },
+            { EmailSenderStringConstants.ExpirationTime, expirationTime },
+        };
+
+        var job = JobBuilder.Create<EmailSenderJob>()
+            .UsingJobData(jobData)
+            .StoreDurably()
+            .Build();
+
+        await scheduler.AddJob(job, false);
+    }
+
+    private string EncodeToBase64(string toEncode)
+    {
+        byte[] toEncodeAsBytes = Encoding.ASCII.GetBytes(toEncode);
```

Comment:
Using ASCII encoding in emails can lead to issues with Ukrainian characters in the content.
Same in `EmailSenderJob`.

Reply:
This was indeed a problem. Thank you for your comment and attention. I fixed it, could you please re-review the code